books/free-programming-books-langs.md

```diff
@@ -1388,6 +1388,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Linux Inside](https://0xax.gitbooks.io/linux-insides/content/index.html)
 * [Linux Kernel in a Nutshell](http://www.kroah.com/lkn/)
 * [Linux Newbie Administrator Guide](http://lnag.sourceforge.net)
+* [Linux Notes For Professionals](https://goalkicker.com/LinuxBook/LinuxNotesForProfessionals.pdf) (PDF)
```

Comment:
Please add the "Compiled from StackOverflow documentation" attribution we use for goalkicker books

Reply:
Sorry I didn't notice, the book is already there but with different name. Thankyou! I must close this PR now.